pkg/asset/imagebased/configimage/installconfig.go

```diff
@@ -2,15 +2,22 @@ package configimage
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"os"
+	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
Can we not introduce this deprecated package in new code?

Reply:
You can use `fmt.Errorf` with the `%w` verb as in other parts of the code for wrapping errors.

Reply:
It was all over the code in installer so i just used it

Reply:
We're trying to slowly replace it as we go.